homeassistant/components/obihai/sensor.py

```diff
@@ -48,6 +48,13 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
         _LOGGER.error("Invalid credentials")
         return
 
+    def _reboot(call):
+        success = pyobihai.call_reboot()
```

Comment:
Let's add back the doc string in the correct place, also make sure to end it with a period like the rest of the functions on this page.  I am not 100% sure if docstrings are required by the core team but since we have it on every function we should make sure it matches.

Reply:
Sure thing.